### DISCUSSION TITLE
Which sport requires a shuttlecock?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#824] [Galaxy Trivia] [answered]
**Bonus:** How many 🪶🪶 does it have?

**Comment by peterbaccaro** [comment, marked as answer]
The sport that requires a shuttlecock is **badminton**.
A traditional badminton shuttlecock typically has **16 feathers**. The feathers are typically goose or duck feathers and are arranged in a circular pattern and attached to a cork base. The feathers provide stability and aerodynamic properties to the shuttlecock during play.

**Comment by SANKARAMDAS** [comment]
Badminton
Badminton is a racket sport played by either two opposing players (singles) or two opposing pairs (doubles), who take positions on opposite halves of a rectangular court that is divided by a net.
